job-utilities/README.MD

```diff
@@ -0,0 +1,60 @@
+# job-utilities
+
+These utilites are used for assisting in job operations in a deployed environment and to a degree replicate job scheduling for local development
+
+## job-details
+
+A core piece for these utilities to work is the job-details.json. This file contains details on a jobs endpoint, what service the job is for, 
+what schedule the job should be run on, etc. The structure of this file is described below.
+
+{
```

Comment:
don't you need to either triple backtick these or put 4 spaces in front of all the code?

Reply:
yep, turns out I do need backticks